weapons-on-back/cl_weapons-on-back.lua

```diff
@@ -39,28 +40,46 @@ local SETTINGS = {
         ["w_lr_firework"] = 2138347493
     }
 }
+--- Toggle Settings ---
+local toggleOn = 0 --- If you want to activate toggle, change value to 1 - 1 for yes, 0 for no.
+local toggle = (1,19) --- Change toggle key here. Currently: left alt.

```

Comment:
Are you sure this doesn't throw any Lua interpreter errors when assigning a variable like this?